w-books/src/App/components/Header/styles.css

```diff
@@ -0,0 +1,27 @@
+.Header {
+  background-color: #FFFFFF;
+  box-shadow: 0 5px 5px #DDDDDD;
+  color: white;
+  height: 96px;
+}
+
+.Header-logo {
+  height: 42px;
+  width: 182px;
+  float: left;
+  position: relative;
+  top: 23px;
```

Comment:
So, using flex dont need specific position and `top/bottom/left/right`
Elements order by flex properties